serving/src/main/java/ai/djl/serving/http/KServeRequestHandler.java

```diff
@@ -174,4 +392,124 @@ private void onException(Exception ex, ChannelHandlerContext ctx) {
 
         NettyUtils.sendJsonResponse(ctx, content, status);
     }
+
+    void onModelException(Throwable t, ChannelHandlerContext ctx) {
+        HttpResponseStatus status;
+        if (t instanceof TranslateException) {
+            status = HttpResponseStatus.BAD_REQUEST;
+        } else if (t instanceof WlmException) {
+            logger.warn(t.getMessage(), t);
+            status = HttpResponseStatus.SERVICE_UNAVAILABLE;
+        } else {
+            logger.warn("Unexpected error", t);
+            status = HttpResponseStatus.INTERNAL_SERVER_ERROR;
+        }
+        if (ctx != null) {
+            NettyUtils.sendError(ctx, status, t);
+        }
+    }
+}
+
+class RequestInput {
+    private String name;
```

Comment:
Will it be possible to reuse KServeIO class?

Reply:
See whether this design would be helpful for you. 

You can move that KServeIO class from inner class to http package. 
Make KServeIO to extend Input.java
And then you can add any other variables that are needed. 
You can methods which would directly convert bytes to the request input or output class you needed.